h/static/scripts/annotator/plugin/textselection.coffee

```diff
@@ -6,13 +6,17 @@ class Annotator.Plugin.TextSelection extends Annotator.Plugin
     @$ = Annotator.$
 
     # Register the event handlers required for creating a selection
-    $(document).bind({
-      "mouseup": @checkForEndSelection
-    })
+    if /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)
```

Comment:
Alternatively, there might be no harm in just binding `checkForEndSelection` to both `touchend` and `mouseup`, regardless of the platform. Worth checking.